README.md

```diff
@@ -2,19 +2,134 @@
+- [ ] Add engine cli integration tests [P0][APPS]
+
+### Online updates
+
+- [ ] Gitbase to support online updates [P1][DP]
+
+### Release necessary language support for enterprise
+
+- [ ] C# driver [P0][LA]
+
+### Get bblfsh ready for wide language support of our use cases
+
+- [ ] Make UAST v2 the standard interface for Babelfish [P0][LA]
+- [ ] Compact diffs of UAST trees [P1][LA]
+- [ ] Plan for a language-independent style-analyzer (bblfsh-side support) [P2][LA]
+- [ ] Provide an efficient storage format for large UASTs collections [P3][LA]
+
 ## Improve the 'Code as Data' individual user experience
 
 ## Improve quality across all products
 
+- [ ] Conduct two empathy sessions for cross-team documentation and user experience improvements [P1][ENG]
+
 ## Release a new version of PGA
 
+- [ ] Release a new version, updated, fixed, including #stars [P1][DR]
+
 ## Bring 'Assisted Code Review' to developers in the open-source community
 
+### Improve current lookout analyzers
+
+- Style Analyzer in one more language (possibly Python) [P0][ML]
+- Release Typo Analyzer [P0][ML]
+
+### Create new lookout analyzers
+
+- [ ] Analyzers for most common linters / traditional static tools [P?][APPS][PRODUCT]
+- [ ] Collect a structured dataset of diffs for best-practices-analyzer [P1][ML]
+- [ ] Research what ML team can deliver for best-practices-analyzer [P1][ML]
+
+### Lookout Platform
+
+- [ ] Collect GitHub reactions as feedback to comments [P1][APPS]
+- [ ] Provide metrics on usage [P0][APPS]
+- [ ] Live Demo Repository with all available analyzers set-up [P1][APPS][INFRA]
+
 ## Store the world's source code
 
-## Improve local-gpu cluster usability
+### Refactor to go-borges library, to reuse borges logic in other projects
+
+- [ ] Library for access to a collection of git repositories (go-borges) [P0][DR]
+- [ ] go-borges-based borges [P0][DR]
+- [ ] siva backend for go-borges [P1][DR]
+
+### Improve borges storage
+
+- [ ] Inspect storage efficiency (review siva usage, siva compaction, git gc, etc) [P1][DR]
+- [ ] Research distributed storage alternatives [P2][INFRA][DR]
+
+## Improve infrastructure
+
+### Release the production kubernetes pipeline cluster
+
+- [ ] Release of the production cluster [P0][INFRA]
+- [ ] Run rovers and borges on it [P0][INFRA]
+- [ ] Deploy services with HA: PostgreSQL, RabbitMQ, etcd [P0][INFRA]
+- [ ] Add monitoring to the pipeline (rovers, borges and associated services) [P0][INFRA]
+- [ ] Migrate staging cluster to the new deployment method [P0][INFRA]
+- [ ] Improve Helm Terraform Provider [P1][INFRA]
+- [ ] New staging cluster with latest Code as Data stack [P1][INFRA]
+
+### Improve local-gpu cluster usability
+
+- [ ] Migrate to the new ML cluster [P0][ML][INFRA]
 
 ## Being a better company to work at
 
 ## Advance MLonCode awareness
 
+- [ ] Publish the final id2vec model [P2][ML]
+
+## Hiring
+
+- Hiring: Test (1 Senior) [P0][ENG]
```

Comment:
No juniors for any team?

Reply:
So far as OKR only for LA team. But we may still hire beyond these.